app/services/doc_auth/errors.rb

```diff
@@ -4,6 +4,8 @@ module Errors
     IMAGE_LOAD_FAILURE = 'image_load_failure' # 438
     PIXEL_DEPTH_FAILURE = 'pixel_depth_failure' # 439
     IMAGE_SIZE_FAILURE = 'image_size_failure' # 440
+    # Network
+    NETWORK = 'network' # usually 500 or other unhandled error
```

Comment:
```suggestion
    NETWORK = 'network_error' # usually 500 or other unhandled error
```

[I think this should be `network_error`](https://github.com/18F/identity-idp/blob/2fd61d8b6633a3a213fba356939fa5b190e1a3a2/config/locales/doc_auth/en.yml#L77).

Reply:
I take that back, I think [I misunderstood what this value did](https://github.com/18F/identity-idp/blob/main/app/services/doc_auth_router.rb#L139).